docs/docs/services/qix-engine/access-control.md

```diff
@@ -4,21 +4,40 @@
     This feature is in an experimental state. Use with caution
     since this feature may change in the future.
 
-Being able to control which users can do what in the context
-of a document is often necessary. In Qlik Associative Engine we support
-[Attribute-Based Access Control (ABAC)](https://en.wikipedia.org/wiki/Attribute-based_access_control)
-which lets you control the document resources based on a users'
-attributes rather than using roles or other, more static, means.
+Being able to control which users can do what in the context of a document is often necessary. In Qlik Associative
+Engine we support [Attribute-Based Access Control (ABAC)](https://en.wikipedia.org/wiki/Attribute-based_access_control)
+which lets you control the document resources based on a user's attributes rather than using roles or other more static
+means.
 
 ## Configuration
 
-Rules are defined, one on each row, in text files. Qlik Associative Engine will
-apply rules in order (except for deny rules which are always applied first,
-regardless of where they are defined).
+### Rules
 
-If the ABAC feature is enabled, all users will by default have no
-access, so it is important that you also include rules when enabling it
-or you will cause access problems for your users.
+Each rule is a conditional expression that evaluates to either `true` or `false`. Typically, a rule asserts if a _User_
+with certain attributes is allowed to perform a specific _Action_ on a certain _Resource_.
+
+ABAC rule evaluation is enabled in the engine by using the `-S EnableABAC=1` command line switch.
+
+!!! Note
+    If the ABAC feature is enabled, all users will by default have no access, so it is important that you also include
+    rules when enabling it or you will cause access problems for your users.
+
+### Rule Files
+
+Rules are defined in text files with one rule on each row. Qlik Associative Engine applies rules in the order they
```

Comment:
It's a bit confusing, but it's not necessary to use the deny file to specify deny rules, they can also be defined in the allow rule set AFAIK.

Reply:
Truly confusing... So you can actually write "deny" rules in the "allow" file that when they evaluate to `true` denies access? I would almost say that it sounds like a bug :-)

Reply:
I think you need to verify most of your questions here with @henriksvenell or @tstenvi as I'm not up-to-date. I believe this was the case previously. Having _one_ file to define your rules seems easier to understand/maintain in my opinion (so maybe drop `Allow` and `Deny` settings and only have a `ABACRuleFile=` setting?)